packages/babel-plugin-transform-class-properties/src/index.js

```diff
@@ -36,7 +36,7 @@ export default function({ types: t }) {
       VALUE: value ? value : t.identifier("undefined"),
```

Comment:
@hzoo regarding linting, would the rule be to _always_ disallow `t.identifier("undefined")` in favor of `path.scope.buildUndefinedNode()`?